FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -68,33 +49,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
     
-    func transitionTo(rootViewController viewController: UIViewController) {
-        guard let window = self.window else { return }
-        guard window.rootViewController != viewController else { return }
-        
-        let snapshot = window.snapshotView(afterScreenUpdates: false)!
```

Comment:
what was this snapshot stuff used for?

Reply:
This was a custom UIKit transition between the welcome screen and the main screen. Not really noticeable, imo, I am interested in how to do the same thing on SwiftUI but not something needed in this location or for launch